### PR TITLE
homework 9: Hibernate

### PR DESCRIPTION
**Pull request by NarboMast**
Implemented functionalities from hw8 using Hibernate.
Not implemented:
Add functionality allows user to update both user and all his tickets in the same transaction.
I did not really understand what i need to do. To update what?

### CODE REVIEW

File: src/main/java/TicketService.java [hunk 1]
```diff
@@ -30,46 +32,58 @@ public static void main(String[] args) {
...
-            System.out.println(busTicket3.toString());
-        }
-
-        User user3 = dao.fetchUserById(2);
-        if (user3 != null) {
-            System.out.println(user3.toPrint());
-        }
-
-        dao.deleteUserAndTicketsById(2);
+//        String url = "jdbc:postgresql://localhost:5432/my_ticket_service_db";
```
Comment (Darya-Drutsko) [inline src/main/java/TicketService.java]: Commented code?

File: src/main/java/busTickets/BusTicket.java [hunk 2]
```diff
@@ -1,16 +1,28 @@
 package busTickets;
 
+import jakarta.persistence.*;
+
 import java.time.LocalDate;
 
+@Entity
+@Table(name = "Ticket")
 public class BusTicket {
+    @Id @GeneratedValue
```
Comment (Darya-Drutsko) [inline src/main/java/busTickets/BusTicket.java]: Put each annotation from new line

File: src/main/java/TicketService.java (hunk 1 above)
Comment (NarboMast) [reply src/main/java/TicketService.java]: commented for a process of development.
Uncommented now

File: src/main/java/busTickets/BusTicket.java (hunk 2 above)
Comment (NarboMast) [reply src/main/java/busTickets/BusTicket.java:10]: Could you please clarify?

Comment (NarboMast) [reply src/main/java/busTickets/BusTicket.java]: done

File: src/main/java/TicketService.java [hunk 3]
```diff
@@ -30,7 +32,6 @@ public static void main(String[] args) {
         BusTicketStorage busTicketStorage = new BusTicketStorage(JsonFilePath);
         System.out.println(busTicketStorage.toString());
 
-
         //Task 8
         String url = "jdbc:postgresql://localhost:5432/my_ticket_service_db";
```
Comment (Darya-Drutsko) [inline src/main/java/TicketService.java:36]: Do we actually need this hardcoded values here? We have already all config in hibernate.cfg.xml

File: src/main/java/users/User.java [hunk 4]
```diff
@@ -1,11 +1,22 @@
 package users;
 
+import jakarta.persistence.*;
+
 import java.time.LocalDate;
 
+@Entity
+@Table(name = "\"User\"")
 public class User {
+    @Id @GeneratedValue
```
Comment (Darya-Drutsko) [inline src/main/java/users/User.java:10]: Put each annotation from new line

**Pull request merged**